admin/admin_info.md

```diff
@@ -0,0 +1,42 @@
+# Admin related topics
+
+## Social media
+
+- nilearn X (twitter) account: https://twitter.com/nilearn
+  - Linked to Nicolas's email
+- nilearn mastodon account: https://fosstodon.org/@nilearn
+  - Linked to Yasmin's email
+
+**Action:** transfer emails, make sure people who need access have login info
+
+## Coredev meeting planning
+
+- Once a month, usually on Tuesdays at 18h CET
+- Jisti link: https://meet.jit.si/nilearn-dev-team-meeting
+- Agenda: https://hackmd.io/hl-vZfHwTkayO5AiT9vvkQ
+
+**Action:** take over of planning
+
+## Drop in hour planning
+
+- Usually every Wednesday at 16h UTC
+- [Current details](https://arewemeetingyet.com/UTC/2023-01-18/16:00/w/Nilearn%20Drop-in%20Hours#eyJ1cmwiOiJodHRwczovL21lZXQuaml0LnNpL25pbGVhcm4tZHJvcC1pbi1ob3VycyJ9)
+- Jitsi link: https://meet.jit.si/nilearn-drop-in-hours
+
+**Action:** take over of planning
+
+## CircleCI token
+
+- Used to trigger circleci hosting of artifacts
+- Personal API token (ref https://circleci.com/docs/managing-api-tokens/)
+  - Generated in Yasmin's account in https://app.circleci.com/settings/user/tokens
+  - Used in https://github.com/nilearn/nilearn/settings/secrets/actions
+
+**Action:** transfer API token
+
+## Other accounts (no action needed)
+
+- https://neurostars.org/tag/nilearn
+- https://pypi.org/project/nilearn/
+- https://discord.gg/vaMvk4NZAx
+- https://drive.google.com/drive/folders/1sTjTyM7ezeWwmbtDvOKCW2H7qFOUIOg9
```

Comment:
Just realized we have a nilearn OSF account and some projects asociated.

Not sure who is behind the account.

Reply:
Good point. It's not me at least ;-)
I'm going to ask Gael about that.